include/mbgl/style/layers/symbol_layer.hpp

```diff
@@ -9,6 +9,7 @@
 #include <mbgl/style/property_value.hpp>
 #include <mbgl/style/expression/formatted.hpp>
 #include <mbgl/util/color.hpp>
+#include <mbgl/util/variable_anchor_offset_collection.hpp>
```

Comment:
@wangyingfang, shouldn't there also be change in layer.hpp.ejs template? Otherwise those changes will be overwritten on next re-gen, right?